requirements.txt

```diff
@@ -0,0 +1,2 @@
+aiohttp==2.2.3
```

Comment:
I don't like this file.
If you need new requirement -- please put it into `requirements` folder.